themes/default/content/case-studies/atlassian.md

```diff
@@ -1,11 +1,10 @@
 ---
-title: Atlassian
+title_tag: Atlassian Bitbucket Cloud | Case Studies
```

Comment:
just calling this out,  but we automatically add a ` | Pulumi`  to the end of all the title tags for pages under this route and the complete title tag ends up being `Atlassian Bitbucket Cloud | Case Studies | Pulumi`. I just want to make sure this is  what we intend since it is not super obvious.

Reply:
Appreciate the callout here! Yes, the complete title tag (with the appended | Pulumi) is the intended outcome.